rlclientlib/federation/local_client.cc

```diff
@@ -84,13 +93,14 @@ int local_client::create(std::unique_ptr<i_federated_client>& output, const util
     i_trace* trace_logger, api_status* status)
 {
   // Create empty model based on ML args on first call
-  std::string initial_command_line(config.get(
-      name::MODEL_VW_INITIAL_COMMAND_LINE, "--cb_explore_adf --json --quiet --epsilon 0.0 --first_only --id N/A"));
+  std::string initial_command_line(config.get(name::MODEL_VW_INITIAL_COMMAND_LINE,
+      "--cb_explore_adf --json --quiet --epsilon 0.0 --first_only --id test_app_id"));
 
   // TODO try catch
   auto args = VW::make_unique<VW::config::options_cli>(VW::split_command_line(initial_command_line));
   auto logger = utility::make_vw_trace_logger(trace_logger);
   auto workspace = VW::initialize_experimental(std::move(args), nullptr, nullptr, nullptr, &logger);
+  workspace->id += "/0";  // initialize iteration id to 0
```

Comment:
Does the id already include app id?

Reply:
its being passed in with the `--id` command in vw. Thinking it might make more sense to include it as an arg in create?

Reply:
Okay cool. I don't follow the arg in create? But i just wanted to double check it was set somewhere else already